commands/activations.go

```diff
@@ -288,31 +297,147 @@ func RunActivationsList(c *CmdConfig) error {
 // RunActivationsLogs supports the 'activations logs' command
 func RunActivationsLogs(c *CmdConfig) error {
 	argCount := len(c.Args)
+
 	if argCount > 1 {
 		return doctl.NewTooManyArgsErr(c.NS)
 	}
-	replaceFunctionWithAction(c)
-	augmentPackageWithDeployed(c)
-	if isWatching(c) {
-		return RunServerlessExecStreaming(activationLogs, c, []string{flagLast, flagStrip, flagWatch, flagDeployed}, []string{flagAction, flagPackage, flagLimit})
+
+	var activationId string
+	if argCount == 1 {
+		activationId = c.Args[0]
+	}
+
+	sls := c.Serverless()
+
+	limitFlag, _ := c.Doit.GetInt(c.NS, flagLimit)
+	stripFlag, _ := c.Doit.GetBool(c.NS, flagStrip)
+	followFlag, _ := c.Doit.GetBool(c.NS, flagFollow)
+	functionFlag, _ := c.Doit.GetString(c.NS, flagFunction)
+	packageFlag, _ := c.Doit.GetString(c.NS, flagPackage)
+
+	limit := limitFlag
+	if limitFlag > 200 {
+		limit = 200
+	}
+
+	if activationId != "" {
+		actv, err := sls.GetActivationLogs(activationId)
+		if err != nil {
+			return err
+		}
+		printLogs(c.Out, stripFlag, actv)
+		return nil
+
+	} else if followFlag {
+		var wg sync.WaitGroup
```

Comment:
I wonder if we might just want to simplify things here and not use the wait group? Wait groups are very useful when you have multiple pieces of work to do in parallel and want to block until they have all completed. In this case, we're only polling for logs. We can remove a bit of complexity by just calling `pollActivations` and blocking until it completes. It would need to be updated to return an error.

Reply:
I made some updates to remove the wait group. I left the other error and signal channel so I can display a cleaner output when users use `control+c`

without it I get the signal interrupted which I didn't like. 
```bash
⇒ go run cmd/doctl/main.go sls activations logs --follow
^Csignal: interrupt
```